apps/ios/Shared/Views/Chat/Group/GroupMemberInfoView.swift

```diff
@@ -52,7 +52,7 @@ struct GroupMemberInfoView: View {
                     .listRowBackground(Color.clear)
 
                 Section {
-                    if let contactId = member.memberContactId {
+                    if let contactId = member.memberContactId, member.memberActive {
```

Comment:
maybe put memberActive condition outside of the section? (it does a bit of whitespace when it's empty)